venue/utils.py

```diff
@@ -71,3 +72,30 @@ def check_language_exists(language_code):
     :return: bool
     """
     return language_code in [l[0] for l in settings.LANGUAGES]
+
+
+# ---------------------------------
+# Constant Contact helper functions
+# ---------------------------------
+
+
+def send_to_constant_contact(username, email):
+    API_KEY = '8ac797fe-583e-4f8b-a0d0-104992af8f9b'
+    ACCESS_TOKEN = '4sahxwfafhy2xy97hq7dgqjz'
```

Comment:
We should not store the access token (or ideally API key) in this public repository.

Reply:
Addressed in my follow up commit. I'll invalidate those API key and access token. I'll create a new set and set it to you in private.